packages/types/index.d.ts

```diff
@@ -353,7 +353,7 @@ interface IServerRenderParams {
   htmlTemplate?: string;
   mountElementId?: string;
   context?: object;
-  mode?: string;
+  mode?: string | Stream;
```

Comment:
看了下模板生成的`server.ts`，里面`render`函数里有这一段解构
```ts
  const {
    path,
    htmlTemplate = '',
    mountElementId = 'root',
    context = {},
    mode = 'string',
    basename = '/',
    staticMarkup = false,
    forceInitial = false,
    getInitialPropsCtx,
  } = params;
  let manifest = params.manifest;
```
看来`mode`属性作为`flag`，类型确实应该是`string`，不会是`Stream`类型。
所以我准备回退提交。
```suggestion
  mode?: string;
```

Reply:
mode?: 'string' | 'stream'

Reply:
```suggestion
  mode?: 'string' | 'stream';
```